README.md

```diff
@@ -288,3 +288,7 @@ Found in `style-dictionary/transforms`
 See API for [creating a transform](https://amzn.github.io/style-dictionary/#/api?id=registertransform)
 
 See [transform docs](https://amzn.github.io/style-dictionary/#/transforms).
+
```

Comment:
any easy way to pass a bool into the build to validate that changes happened purposely - then automate the rebuild of validate-structure?

Reply:
Added in!

Reply:
Running `npm run validate -- --update` will do the trick